python/EmpireWrapper.cpp

```diff
@@ -324,6 +324,8 @@ namespace FreeOrionPython {
 
             .def("policyAvailable",                 +[](const Empire& e, const std::string& policy) { return e.PolicyAvailable(policy); })
 
+            .def("policyPrereqsAndExclusionsOK",    +[](const Empire& e, const std::string& policy) { return e.PolicyPrereqsAndExclusionsOK(policy); })
```

Comment:
Maybe name it  `policyCouldBeAdopted()`

Reply:
Well, I simply stuck to the custom of forwarding the C++ function to a python function with the same name. Plus couldBeAdopted sounds like it also checks for required IP, the current name specifies exactly what it is doing.

Reply:
The current name is probably better as it's more specific. As @Grummel7 notes, whether a policy can be adopted is more than that its prereqs and exclusions are satisfied.